webpack/assets/javascripts/react_app/components/Loading/Loading.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Bullseye,
+  Title,
+  EmptyState,
+  EmptyStateIcon,
+  Spinner,
+} from '@patternfly/react-core';
+
+// Centered patternfly 4 loading icon
+const Loading = ({ textSize, iconSize, showText }) => (
+  <Bullseye>
+    <EmptyState>
+      <EmptyStateIcon
+        variant="container"
+        component={() => <Spinner size={iconSize} aria-label="loading icon" />}
+      />
+      {showText && (
+        <Title size={textSize} headingLevel="h4">
+          {__("Loading")}
```

Comment:
Shouldn't we have a way to replace the text using prop?

Reply:
I don't see a need for it now, it is a pretty easy addition if we ever have a use case, we can add with a `Loading` default value.

I tend to lean a bit more towards the [YAGNI](https://www.martinfowler.com/bliki/Yagni.html) philosophy when it comes to these kind of things.